packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Configure Device Authorization Grant Flow
+excerpt: Learn how to use a secondary device to complete sign-in to applications
+layout: Guides
+---
+
+<ApiLifecycle access="ea" /><br>
+
+## Overview
+
+The Device Authorization feature is an OAuth 2.0 grant type. It allows users to sign in to input-constrained devices, such as smart TVs, digital picture frames, and printers, as well as devices with no browser. Device Authorization enables you to use a secondary device, such as a laptop or mobile phone, to complete sign-in to applications that run on such devices.
+
+The Device Authorization feature is available for both Okta Classic and Okta Identity Engine orgs.
+
+If you need help or have an issue, post a question on the [Okta Developer Forum](https://devforum.okta.com).
+
+## Before you begin
+
+This guide assumes that you:
+
+* Have an Okta Developer Edition organiztion. Don't have one? [Create one for free](https://developer.okta.com/signup).
+* Have the Native SSO feature enabled for your org. From the left navigation pane in the Admin Console, go to **Settings** > **Features**, locate the Native SSO slider, and slide to enable.
```

Comment:
This is wrong, it's needs the Device Authorization Grant, not Native SSO

Reply:
Fixed